neovim/init.vim

```diff
@@ -0,0 +1,4 @@
+echo "Enter string to encode:\n"
+no <silent> \rot13 g?G
+no <silent> \prepend OEncoded string:<Esc>
```

Comment:
should there be a `0`? :pleading_face:

Reply:
No, because it prepends the line to _current_ string, and the current string is the first one (see `actions`)

Reply:
meow